data/testimonials/5.yaml

```diff
@@ -1,4 +1,4 @@
-text: "“For pioneering the use of machine learning to assist developers in generating code snippets, paving the way for today’s intelligent code assistance tools.”"
+text: "“For providing foundational understanding of modern code review, advancing the state of practice, and charting the course of research in code review through empirical investigations.”"
 name: "Mira Mezini"
```

Comment:
this code review paper wasn't from Mira, it was a BachelliBird, right? https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/ICSE202013-codereview.pdf